ARM.CMSIS.pdsc

```diff
@@ -2490,6 +2490,7 @@ and 8-bit Java bytecodes in Jazelle state.
       <files>
         <!-- include folder / device header file -->
         <file category="include"      name="Device/ARM/ARMCM55/Include/"/>
+        <file category="include"      name="Device/ARM/ARMCM55/Include/Template"/>
```

Comment:
This should not be required and may even break projects.
The `partition_ARMCM55.h` file is declared as a config file below. Hence is needs to be copied into the user project to be modifiably.

Reply:
Sorry Jonatan, it was a secure/non-secure mismatch in my project.
I'll close the PR.